### PR TITLE
worked on the initial curriculum database, create/edit pages, and creati...

### PR DESCRIPTION
**Pull request by seth-marquez**
...ng a description for the curricula

### CODE REVIEW

File: app/controllers/curricula_controller.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	#where to put the user to auto assign the creater/owner
+	def load
+
+	end
+
+	def create
+		if request.post?
+			@info = Curricula.new(post_params)
+			@info.save
+			redirect_to "/dashboard/dashboard_main"
```
Comment (bammons) [inline app/controllers/curricula_controller.rb]: would prefer this was url variable
should be something like: dasboard_dashboard_main_path or dasboard_dashboard_main

File: app/controllers/curricula_controller.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	end
+
+	def edit
+		@info = Curricula.all
+	end
+
+	private
+
+	def post_params
+		params.permit(:cur_name, :cur_description)
```
Comment (kdorer) [inline app/controllers/curricula_controller.rb]: should require a name for the curriculum

File: app/controllers/curricula_controller.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+class CurriculaController < ApplicationController
+	#where to put the user to auto assign the creater/owner
+	def load
+
+	end
+
+	def create
+		if request.post?
```
Comment (bammons) [inline app/controllers/curricula_controller.rb:8]: I think this should look like:
Curricula.create(params[:curricula])

File: app/controllers/curricula_controller.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+			@info = Curricula.new(post_params)
+			@info.save
+			redirect_to "/dashboard/dashboard_main"
+		end
+	end
+
+	def edit
+		@info = Curricula.all
```
Comment (bammons) [inline app/controllers/curricula_controller.rb]: And this should look more like the one up there:
#code should be close but doesn't actually work
curricula = current_account.curriculas.find(params[:id])
person.update_attributes!(curricula_params)
redirect_to dashboard_path

File: app/controllers/curricula_controller.rb (hunk 2 above)
Comment (bammons) [inline app/controllers/curricula_controller.rb]: And finally:
params.require(:curricula).permit(:cur_name, :cur_description)

**Comment by bammons** [comment]
-1 needs a few changes

**Comment by kdorer** [comment]
-1 fix the stuff and things

File: app/controllers/curricula_controller.rb [hunk 5]
```diff
@@ -0,0 +1,27 @@
+class CurriculaController < ApplicationController
+	#where to put the user to auto assign the creater/owner
+	def load
+
+	end
+
+	def create
+		if request.post?
+			@curricula = Curricula.new(curricula_params)
```
Comment (seth-marquez) [inline app/controllers/curricula_controller.rb:9]: @bammons used new and save instead of create for semantic reasons

**Comment by bammons** [comment]
I'm going to +1 this and do the update dashboard and edit tasks

**Comment by seth-marquez** [comment]
solved #25

**Pull request merged**